elide-async/src/main/java/com/yahoo/elide/async/models/AsyncQuery.java

```diff
@@ -69,8 +68,7 @@
     private QueryStatus status = QueryStatus.QUEUED;
 
     @Enumerated(EnumType.STRING)
-    @NotNull
```

Comment:
Why remove NotNull?

Reply:
I thought since it's going to have defaulted value anyways, a check for NotNull could be skipped.